### ISSUE TITLE
Streaming audio in skills kit

### ISSUE THREAD

**Issue by sweenzor** [paszin/soundcloud_alexa_bridge#1]
(Not really an issue, just FYI, thanks for sharing this project!)

https://developer.amazon.com/blogs/post/Tx1DSINBM8LUNHY/New-Alexa-Skills-Kit-ASK-Feature-Audio-Streaming-in-Alexa-Skills

**Comment by rocketinventor** [comment]
@paszin I was thingking about that, too. It would be great if we could use this feature. It would make the setup a lot easier.

**Comment by paszin** [comment]
@sweenzor You are right. Unfortunately this feature was released after I build this project. 
But I agree, it would be nice :)

**Comment by rocketinventor** [comment]
@paszin Is this a planned feature, then?

**Comment by paszin** [comment]
Finally, I created a new repository leveraging the integrated audio player.

https://github.com/paszin/alexa-soundcloud

**Comment by sweenzor** [comment]
Whaaaaa! That rules!! Awesome work!
